pkg/utils/json_test.go

```diff
@@ -0,0 +1,148 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+
+	. "github.com/pingcap/check"
+)
+
+type testJSONSuite struct{}
+
+var _ = Suite(&testJSONSuite{})
+
+var testMetaJSONs = [][]byte{
+	[]byte(`{
+  "files": [
+    {
+      "sha256": "aa5cefba077644dbb2aa1d7fae2a0f879b56411195ad62d18caaf4ec76fae48f",
+      "start_key": "7480000000000000365f720000000000000000",
+      "end_key": "7480000000000000365f72ffffffffffffffff00",
+      "name": "1_2_29_6e97c3b17c657c4413724f614a619f5b665b990187b159e7d2b92552076144b6_1617351201040_write.sst",
+      "end_version": 423978913229963260,
+      "crc64xor": 8093018294706077000,
+      "total_kvs": 1,
+      "total_bytes": 27,
+      "cf": "write",
+      "size": 1423
+    }
+  ],
+  "schemas": [
+    {
+      "table": "{\"id\":54,\"name\":{\"O\":\"test\",\"L\":\"test\"},\"charset\":\"utf8mb4\",\"collate\":\"utf8mb4_bin\",\"cols\":[{\"id\":1,\"name\":{\"O\":\"pk\",\"L\":\"pk\"},\"offset\":0,\"origin_default\":null,\"origin_default_bit\":null,\"default\":null,\"default_bit\":null,\"default_is_expr\":false,\"generated_expr_string\":\"\",\"generated_stored\":false,\"dependences\":null,\"type\":{\"Tp\":15,\"Flag\":4099,\"Flen\":256,\"Decimal\":0,\"Charset\":\"utf8mb4\",\"Collate\":\"utf8mb4_bin\",\"Elems\":null},\"state\":5,\"comment\":\"\",\"hidden\":false,\"change_state_info\":null,\"version\":2}],\"index_info\":[{\"id\":1,\"idx_name\":{\"O\":\"PRIMARY\",\"L\":\"primary\"},\"tbl_name\":{\"O\":\"\",\"L\":\"\"},\"idx_cols\":[{\"name\":{\"O\":\"pk\",\"L\":\"pk\"},\"offset\":0,\"length\":-1}],\"state\":5,\"comment\":\"\",\"index_type\":1,\"is_unique\":true,\"is_primary\":true,\"is_invisible\":false,\"is_global\":false}],\"constraint_info\":null,\"fk_info\":null,\"state\":5,\"pk_is_handle\":false,\"is_common_handle\":true,\"common_handle_version\":1,\"comment\":\"\",\"auto_inc_id\":0,\"auto_id_cache\":0,\"auto_rand_id\":0,\"max_col_id\":1,\"max_idx_id\":1,\"max_cst_id\":0,\"update_timestamp\":423978913176223755,\"ShardRowIDBits\":0,\"max_shard_row_id_bits\":0,\"auto_random_bits\":0,\"pre_split_regions\":0,\"partition\":null,\"compression\":\"\",\"view\":null,\"sequence\":null,\"Lock\":null,\"version\":4,\"tiflash_replica\":null,\"is_columnar\":false}",
+      "db": "{\"id\":1,\"db_name\":{\"O\":\"test\",\"L\":\"test\"},\"charset\":\"utf8mb4\",\"collate\":\"utf8mb4_bin\",\"state\":5}",
```

Comment:
is it possible to further decode this as a JSON object?

Reply:
Maybe `map[string]interface{}` could help. Let me try. 🤔

Reply:
Done, pal